### PR TITLE
feat(ui,api): disable ascode edit for template generated components

### PR DESCRIPTION
**Pull request by richardlt**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#13043.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✘

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#13051.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✘

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#13052.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔ 
* Package
  * Docker ✔ 
  * Docs ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✘

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#13068.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔

File: engine/api/templates.go [hunk 1]
```diff
@@ -373,6 +373,7 @@ func (api *API) postTemplateApplyHandler() service.Handler {
 			},
 		}
 
+		//
```
Comment (fsamin) [inline engine/api/templates.go:376]: :100:

File: ui/src/app/service/authentication/logout.interceptor.ts [hunk 2]
```diff
@@ -33,13 +33,19 @@ export class LogoutInterceptor implements HttpInterceptor {
 
                         this._router.navigate(['/auth/signin'], navigationExtras);
                     } else if (req.url.indexOf('auth/me') === -1) { // ignore error on auth/me used for auth pages
+                        console.log(e)
```
Comment (sguiheux) [inline ui/src/app/service/authentication/logout.interceptor.ts:36]: need?

File: ui/src/app/views/pipeline/show/pipeline.show.component.ts [hunk 3]
```diff
@@ -70,7 +69,7 @@ export class PipelineShowComponent implements OnInit {
     queryParams: Params;
     @ViewChild('paramWarning', { static: false })
     parameterModalWarning: WarningModalComponent;
-    @ViewChild('updateEditMode', { static: false})
+    @ViewChild('updateEditMode', { static: false })
```
Comment (sguiheux) [inline ui/src/app/views/pipeline/show/pipeline.show.component.ts:72]: rebase with masteR?  no need static:false with angular 9

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#13113.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔ 
* Package
  * Docker ✔ 
  * Docs ✔ 
* Integration
  * Docker Compose Tests ✘  
  * Standalone Tests ✘

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#13141.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✘  
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔

**Pull request merged**